Source/WebCore/Modules/WebGPU/GPUQueue.cpp

```diff
@@ -54,13 +54,11 @@ void GPUQueue::setLabel(String&& label)
 
 void GPUQueue::submit(Vector<RefPtr<GPUCommandBuffer>>&& commandBuffers)
 {
-    Vector<std::reference_wrapper<WebGPU::CommandBuffer>> result;
-    result.reserveInitialCapacity(commandBuffers.size());
-    for (const auto& commandBuffer : commandBuffers) {
-        if (!commandBuffer)
-            continue;
-        result.uncheckedAppend(commandBuffer->backing());
-    }
+    auto result = WTF::compactMap(commandBuffers, [](auto& commandBuffer) -> std::optional<std::reference_wrapper<WebGPU::CommandBuffer>> {
+        if (commandBuffer)
```

Comment:
Is there a benefit to having the 'work' done in the early return, rather than returning early with `nullopt`? No need to change the code, I'm just curious if there is a benefit to this pattern. It looked strange given that it flips the order of the original code.

Reply:
No benefit. No sure why I changed the order